src/azure-cli/azure/cli/command_modules/vm/_client_factory.py

```diff
@@ -143,3 +143,10 @@ def cf_log_analytics_data_plane(cli_ctx, _):
 
 def cf_disk_encryption_set(cli_ctx, _):
     return _compute_client_factory(cli_ctx).disk_encryption_sets
+
+
+def _dev_test_labs_client_factory(cli_ctx, *_):
```

Comment:
curious why devtestlab client inside vm

Reply:
This feature is implemented using devtestlab service.